src/main/java/com/hotels/beeju/HiveMetaStoreJUnitRule.java

```diff
@@ -96,10 +83,10 @@ public HiveMetaStoreJUnitRule(String databaseName, Map<String, String> configura
 
   @Override
   protected void beforeTest() throws Throwable {
-    final HiveConf hiveConf = new HiveConf(conf, HiveMetaStoreClient.class);
+    final HiveConf hiveConf = new HiveConf(core.conf(), HiveMetaStoreClient.class);
     ExecutorService singleThreadExecutor = Executors.newSingleThreadExecutor();
```

Comment:
Same here this logic can maybe go to some: `HiveMetastoreCore` class, so the rule just delegates.